### DISCUSSION TITLE
Can't get ytdl-core to work.

### DISCUSSION THREAD

**Discussion by TheRooter1** [fent/node-ytdl-core#1234] [Q&A]
Hi.

I've been trying to get the title of a YT video from an URL but so far with no success. Here is my code:

`const ytdl = require("ytdl-core");

ytdl.getBasicInfo('https://www.youtube.com/watch?v=gocwRvLhDf8', function(err, info) {
  console.log(info.title);
});`

I've tried this with the newest version of ytdl-core (4.11.4) on both newest stable version of NodeJS (18.16.0) and an older version of NodeJS (v12.16.3). I get neither a response nor an error. Please help.

**Comment by nelliott82** [comment]
`getBasicInfo`'s second argument is expected to be an options object. I don't believe a callback function as the second argument will get executed.
